src/daemon/utils_cache.c

```diff
@@ -920,23 +920,23 @@ static meta_data_t *uc_get_meta(metric_t const *m) /* {{{ */
     int ret = wrap_function(meta, key);                                        \
     pthread_mutex_unlock(&cache_lock);                                         \
     return ret;                                                                \
-  }
-
-int uc_meta_data_exists(metric_t const *m, const char *key)
-    UC_WRAP(meta_data_exists);
-
-int uc_meta_data_delete(metric_t const *m, const char *key)
-    UC_WRAP(meta_data_delete);
+  } while (0)
 
+int uc_meta_data_exists(metric_t const *m, const char *key) {
+  UC_WRAP(meta_data_exists);
+}
+int uc_meta_data_delete(metric_t const *m, const char *key) {
+  UC_WRAP(meta_data_delete);
+}
 /* The second argument is called `toc` in the API, but the macro expects
  * `key`. */
-int uc_meta_data_toc(metric_t const *m, char ***key) UC_WRAP(meta_data_toc);
+int uc_meta_data_toc(metric_t const *m, char ***key) { UC_WRAP(meta_data_toc); }
 #undef UC_WRAP
 
 /* We need a new version of this macro because the following functions take
- * two argumetns. gratituous semicolons added for formatting sanity*/
+ * two arguments. Gratuitous semicolons added for formatting sanity. */
```

Comment:
With the use of do … while in the macros, the semicolons are now required. Please remove the last sentence.

Reply:
`collectd-bot` is still not working correctly, as it merged this PR after you added `automerge` tag, although this comment was still unresolved. :-/

I could could do another PR fixing some other GCC warnings, and fix that comment in it...

Reply:
Yeah, I'll look into the bot. I think GitHub now has a native automerge feature. Maybe we should enable that instead.

Don't worry about it; I don't think this is worth a separate PR.